helpers.lua

```diff
@@ -808,6 +837,16 @@ function edit_user()
     end
 end
 
+-- hash the user password using sha-512 and using {CRYPT} to uses linux auth system
+-- because ldap doesn't support anything stronger than sha1
+function hash_password(password)
+    -- TODO is the password checked by regex? we don't want to
+    -- allow shell injection
+    local mkpasswd  = io.popen("mkpasswd --method=sha-512 '" ..password.."'")
```

Comment:
Yes it does ... and yes, we don't have a choice right now :(
It happens very unfrequently and tbh, we already store in permanence the user passwords un nginx memory cache to reconnect to ldap if we lose connection -_-' (I didn't managed to find an acceptable way to fix that)